plugin/src/main/java/net/dzikoysk/funnyguilds/user/UserManager.java

```diff
@@ -139,10 +137,6 @@ public boolean playedBefore(String nickname, boolean ignoreCase) {
         return usersByName.containsKey(nickname);
     }
 
-    public int countUsers() {
```

Comment:
Czemu to wylatuje

Reply:
To jedynie zmieniło lokalizacje w klasie bo dalej ta metoda istnieje.

Reply:
Czyli znowu zmieniłeś bez sensu miejsce metody, prosze tak nie robic